stdlib/SparseArrays/test/sparse.jl

```diff
@@ -1803,6 +1803,18 @@ end
     resize!(foo.nzval, 5)
     setindex!(foo.nzval, NaN, 5)
     @test norm(foo) == 2.0
+
+    # Test (m x 1) sparse matrix
+    colM = sprandn(10, 1, 0.6)
+    @test opnorm(colM, 1) ≈ opnorm(Array(colM), 1)
+    @test opnorm(colM) ≈ opnorm(Array(colM))
+    @test opnorm(colM, Inf) ≈ opnorm(Array(colM), Inf)
+
+    # Test (1 x n) sparse matrix
+    rowM = sprandn(1, 10, 0.6)
+    @test opnorm(rowM, 1) ≈ opnorm(Array(rowM), 1)
+    @test opnorm(rowM) ≈ opnorm(Array(rowM))
+    @test opnorm(rowM, Inf) ≈ opnorm(Array(rowM), Inf)
```

Comment:
I guess we should also have one test each for the case of `p` some non-standard number (not 1, 2, `Inf`).